File: CCF110-Programacao/Lista-de-exercicios-04-CCF110/Exercicio12.py
horas = input("Digite as horas, e os minutos, separados por um espaço").split(" ")
hora = int(horas[0])
minutos = int(horas[1])
totalminutos = (hora*60)+minutos

print("Desde as 00:00, se passaram",totalminutos,"minuto(s)")

File: CCF110-Programacao/Lista-de-exercicios-09-CCF110/Exercicio12.c
//Exercicio12
#include <stdio.h>

int x(h,m,s){
    int soma,aux;
    soma=0;
    soma =soma+ h*3600;
    soma = soma + m*60;
    soma = soma+s;
    return(soma);
}

int main(){
    int h,m,s,a;
    printf("Digite as horas, minutos e segundos, separados por um espaço: ");
    scanf("%d %d %d",&h,&m,&s);
    a=x(h,m,s);
    printf("%d",a);
return 0;
}
